draft-ietf-rtcweb-jsep.xml

```diff
@@ -1720,7 +1720,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <xref target="RFC4568"></xref> MUST NOT be implemented to
             signal SDES SRTP keying information.</t>
 
-            <t>The
```

Comment:
I see, now that you mention it.

We could say "RFC 5888 MUST be implemented", i.e. remove the "grouping framework" phrase.